addons/website/static/src/xml/track_page.xml

```diff
@@ -3,7 +3,7 @@
 
 	<t t-name="website.track_page">
 		<div role="separator" class="dropdown-divider"/>
-		<a href="#" name="switch-track-page" class="dropdown-item" role="menuitem">
+		<a name="switch-track-page" class="dropdown-item" role="menuitem">
```

Comment:
For information / validation: @qsm-odoo @JKE-be

Reply:
I guess, don't really know about the refactoring which "broke" `<a href="#"`. Personally, I try not to use those anyway anymore but instead use `<button type="button"` which is more explicit than "this is a link but not really, it act as a button" :wink: But anything is fine here :+1:

Reply:
imo it is the wrong fix.

We know that bootstrap make some shit with link without href, probably working luckily in your case thanks to the class dropdown-item.

You should instead avoid to pass into the code that catch your click (the preventDefault from scrollerService) adding disable_anchor attribute for example and check with @sdegueldre that make the owl conversion, if he can take a more general look to disable it in frontend and avoid future similar bug.

Reply:
I this case I'm pretty sure we want to call preventDefault on this event: you don't want the page to scroll back to the top when you click on the menu-item. These customize items are the only ones that are broken because they rely on the change event from the checkbox, which will not trigger because of preventDefault. I think this fix is ok, I'm not even sure why these are `<a>` in the first place, they should be `<li>` (and there already is CSS to give it a pointer cursor when it's an li).

Reply:
> they should be `<li>` (and there already is CSS to give it a pointer cursor when it's an li).

Please no :fearful: If this is an interactive item, it should be a link or a button. And a `<li>` certainly cannot have the dropdown-item class.

Reply:
EDIT: @sdegueldre Ok you are right, I spoke too fast then opened a runbot: we really do have a link > label > checkbox structure :fearful: -> So yes those should `<li>` but not with the dropdown-item class, definitely some refactoring to do if we want to make it right around here :grimacing:

Reply:
Remember we are talking about 14.5 (aka minimal diff). So it seems fix is ok according to @sdegueldre , waiting for a cleaning to schedule in master.

Reply:
please stop this bullshit reply... 14.5 is a future major release without anybody on it for now and any custo.
So if we can make something clean for the next 3 years it is now and not after the release :/
In this case it is a detail, but I hope that you don't think like that for all fixes in s14.5 :/

Reply:
Thanks for your input.

Reply:
yes, I hope...
I take my time to identify the initial change that break it... And ping the right person... Let me think it is 'really' helpful...
and we are all on the same boat with the purpose to have nice product, including the v15.

It was not a nasty comment, just a way to say that we can still fix properly the saas-14.5 to avoid to have to support wrong design during 3 years.

:kissing_closed_eyes:

Reply:
Actually it was understood as a nasty comment but I understand what is behind :D . I always try to balance minimal diff in stable / refactoring in master (even 14.5, it is frozen even if we can make more changes). Here it seems it is not that much work to do cleanly indeed, so we will try the clean approach :) .